src/index.ts

```diff
@@ -47,10 +47,21 @@ export interface BlockchainOptions {
   db?: any
 
   /**
-   * This the flag indicates if blocks should be validated (e.g. Proof-of-Work), latest HF rules
-   * supported: `Petersburg`.
+   * This the flag indicates if blocks and Proof-of-Work should be validated. Defaults to true.
+   * See
    */
   validate?: boolean
+
+  /**
+   * This flags indicates if Proof-of-work should be validated. Defaults to the value of `validate`.
+   */
+  validatePow?: boolean
+
+  /**
+   * This flags indicates if blocks should be validated. See Block#validate for details. Defaults to
+   * the value of `validate`.
+   */
+  validateBlocks?: boolean
```

Comment:
I am not happy with the redundancy introduced by this sublevel of validation along with the general `validation` flag. On the edge cases (1. `validation` to `true`, both other to `false`, 2. `validation` to `false`, both other to `true`) it's just not really decidable or logically derivable which setting super-sedes and will likely lead to erroneous usage.

My first thinking was... 

1. ...that this would be just worth a breaking change and we should drop the `validate` flag altogether, all cases can be disambibuously covered by this.

Then I thought...

2. on the lower end this could be covered by clearly document by saying which parameter hierarchy takes precedence.

A combination of this would be...

3. To do 2. and along mark the `validate` flag as `DEPRECATED`, to get to some less error prone implementation again on the next breaking release.

What do you think?

Reply:
I like your proposal. I didn't want to remove `validate` just yet, as it's a breaking change, as it's a breaking change. But flagging it as deprecated is a step forward. I'll also improve the documentation.

Reply:
Could you also add the false value defaults as a remark to the docs? (Did you actually remove this along the way, thought this was already in?)